scripts/test_line_bot.py

```
@@ -0,0 +1,233 @@
+    except Exception as e:
+        print(f"❌ 初期化エラー: {e}")
+        return None, None
+    
+    finally:
+        if 'mongo_client' in locals():
+            await mongo_client.disconnect()
+
+
+async def test_message_processing():
+    """メッセージ処理のテスト"""
+    print(f"\n📝 メッセージ処理テスト")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)